yoti_python_sdk/http.py

```diff
@@ -24,12 +24,37 @@
 
 
 class YotiResponse(object):
-    def __init__(self, status_code, text):
+    def __init__(self, status_code, text, headers=None):
+        if headers is None:
+            headers = {}
+
         self.status_code = status_code
         self.text = text
+        self.headers = headers
+
+
+class MediaResponse(object):
```

Comment:
Could we pass `content` into `YotiResponse` if that's the only difference? (then we don't need `MediaResponse`)
Or perhaps it could be named `BinaryResponse`?

Is it possible to have `MediaResponse`/`BinaryResponse` extend `YotiResponse` making use of the existing properties: `status_code`, `text`, `headers`, so it will only have the additional `content` property?

Reply:
Fixed in 6c39f5b